Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "rill"
+version = "0.1.0"
+authors = ["Remi Attab <remi.attab@gmail.com>"]
+license = "2-Clause BSD"
```

Comment:
I don't think you care about this, but the license should come from [this whitelist](https://spdx.org/licenses/) if you care about publishing on crates.io.
I believe it should be `BSD-2-Clause`.